prometheus/operator/base/cluster-metrics-servicemonitor.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: cluster-monitor-federation
+  labels:
```

Comment:
We will need the following label on this service monitor
```
        team: opendatahub
```

Reply:
added :+1: